sys/net/application_layer/nanocoap/nanocoap.c

```diff
@@ -30,6 +30,15 @@
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
+/**
+ * @name    Internally used CoAP packet types
+ * @{
+ */
+#define COAP_REQ                (0)
```

Comment:
Yes, good idea to hide them in the implementation file. Only the COAP_TYPE_xxx constants should be publicly accessible.